datastore/tests/test_additional_data_license.py

```diff
@@ -0,0 +1,37 @@
+from django.test import TestCase
+
+from additional_data.sources.grant_metadata import GrantMetadataSource
+from db.models import Grant
+
+
+class TestAdditionalDataLicense(TestCase):
+    fixtures = ["test_data.json"]
+
+    def test_publisher_license(self):
+        grant_metadata_source = GrantMetadataSource()
+
+        grant = Grant.objects.first()
+        additional_data = {}
+
+        expected_additional_data = {
+            "metadata": {
+                "source_license_name": "Creative Commons Attribution 4.0",
+                "source_license": "https://creativecommons.org/licenses/by/4.0/",
+            }
+        }
+
+        grant_metadata_source.update_additional_data(
+            grant.data, additional_data, source_data=grant.source_file.data
+        )
+
+        self.assertEqual(
+            GrantMetadataSource.ADDITIONAL_DATA_KEY in additional_data,
+            True,
+            "Additional data was not added.",
```

Comment:
minor - Great to have assert messages, i think it would be improved if it said the more about this specific source, as there are many "Additional data" that it could be referring to. e.g. Adding "License additional data was not added" . (Minor because it should be able to be inferred from the class name, just when scanning down a bunch of test outputs it's a useful context reminder.)